package.json

```
@@ -54,6 +54,7 @@
     "mock-socket": "8.0.5",
     "nock": "10.0.6",
     "react": "16.8.6",
+    "redux-mock-store": "^1.5.3",
```

```suggestion
    "redux-mock-store": "1.5.3",
```
We don't auto-update any dependencies

---

once you do this you'll need to update the package-lock.json as well